source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,316 @@
+Deploy Operator With Helm
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+Helm is a tool for automating the deployment of applications to Kubernetes clusters.
+A `Helm chart <https://helm.sh/docs/topics/charts/>`__ is a set of YAML files, templates, and other files that define the deployment details.
+The following procedure uses a Helm Chart to install the :ref:`MinIO Kubernetes Operator <minio-operator-installation>` to a Kubernetes cluster.
+
+
+Prerequesites
+-------------
+
+To install the Operator with Helm you will need the following:
+
+* An existing cluster using Kubernetes 1.19.0 or later (1.21.0 or later recommended).
+* The ``kubectl`` CLI tool on your local host, the same version as the cluster.
+* `Helm <https://helm.sh/docs/intro/install/>`__ version xx or greater.
+* `yq <https://github.com/mikefarah/yq/#install>`__ version xx or greater.
+* Access to run ``kubectl`` commands on the cluster from your local host.
+
+For more about Operator installation requirements, including TLS certificates, see the :ref:`Operator deployment prerequisites <minio-operator-prerequisites>`.
+
+This procedure assumes familiarity the with referenced Kubernetes concepts and utilities.
+While this documentation may provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kuber\
+netes Documentation <>`.
+
+
+Procedure
+---------
+
+
+Install Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Download the Helm charts
+
+   On your local host, download the Operator and tenant Helm charts to a convenient directory:
+
+   .. code-block:: shell
+      :class: copyable
+      :substitutions:
+
+      curl -O https://raw.githubusercontent.com/minio/operator/master/helm-releases/operator-|operator-version-stable|.tgz
+      curl -O https://raw.githubusercontent.com/minio/operator/master/helm-releases/tenant-|operator-version-stable|.tgz
+
+#. Deploy Operator
+
+   The following Helm command deploys the MinIO Operator using the downloaded chart:
+
+   .. code-block:: shell
+      :class: copyable
+
+      helm install \
+      --namespace minio-operator \
+      --create-namespace \
+      minio-operator operator-|operator-version-stable|.tgz
+
+#. Configure the MinIO deployment
+
+   #. Create the YAML configuration files
+   
+      Use ``kubectl`` and ``yq`` to create the following files:
+
+      ``service.yaml``:
+
+      .. code-block:: shell
+         :class: copyable
+
+         kubectl get service console -n minio-operator -o yaml > service.yaml
+         yq e -i '.spec.type="NodePort"' service.yaml
+         yq e -i '.spec.ports[0].nodePort = PORT_NUMBER' service.yaml
+
+      Replace ``PORT_NUMBER`` with the port on which to serve the Operator GUI. 
+
+      The file contents resemble the following:
+
+      .. dropdown:: Example ``service.yaml`` file
```

Comment:
Seems important to show example output, but these yq output examples are long. Hence dropdown. Ideas?

Reply:
I think the dropdowns are a good solution for this.